src/Makefile

```diff
@@ -2,10 +2,13 @@
 # Use `FC=<compiler> ... make` to select compiler
 # Use `DEBUG=1 ... make` for a debug build
 # Use `NETCDF=1 ... make` to build with NetCDF using `nf-config`
+# compile with either FC=ifort or FC=gfortran
 
 # Compiler
 FC ?= gfortran
 ifeq ($(FC), f77)  # override possible Make default
+  FC := ifort 
```

Comment:
We could continue to have `gfortran` here I would think, but @drnimbusrain can decide.

Reply:
Yes, lets leave gfortran here.

Reply:
@zmoon @drnimbusrain , i think we need this change, When i revert back, FC shows as gfortran and i get build error:
Fatal Error: File 'netcdf.mod' opened at (1) is not a GNU Fortran module file
make: *** [canopy_ncf_io_mod.o] Error 1
I think we need these changes (or FC to be assigned ifort somewhere)